spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,72 @@
+feature 'FEATURE: sign up' do
+
+  scenario 'Can click to sign up from home page' do
+    visit('/')
+    click_link('sign_up')
+    expect(page.current_path).to eq "/users/new"
+  end
+
+  let(:name) { 'J T Kirk' }
+  let(:email) { 'JTK@test.com' }
+  let(:user_name) { 'TheShat' }
+  let(:password) { 'password' }
+  let(:password_confirmation) { 'password' }
```

Comment:
Yes I thought about this for a while, and figured that if I extracted them then it would be much harder to understand the tests, so decided to leave them :)